packages/client/hmi-client/src/components/model/petrinet/tera-model-description.vue

```diff
@@ -5,71 +5,616 @@
 				This page describes the model. Use the content switcher above to see the diagram and manage
 				configurations.
 			</Message>
-			<table>
+			<table class="bibliography">
 				<tr>
 					<th>Framework</th>
 					<th>Model version</th>
 					<th>Date created</th>
 					<th>Created by</th>
 					<th>Source</th>
+					<th>Author email</th>
+					<th>Institution</th>
+					<th>License</th>
+					<th>Complexity</th>
 				</tr>
 				<tr>
-					<td class="framework">{{ model?.header.schema_name }}</td>
-					<td>{{ model?.header.model_version }}</td>
-					<td>{{ model?.metadata?.processed_at }}</td>
-					<td>{{ model?.metadata?.annotations?.authors?.join(', ') }}</td>
+					<td class="framework">{{ model?.header?.schema_name }}</td>
+					<td>{{ model?.header?.model_version }}</td>
+					<td>{{ model?.metadata?.processed_at ?? card?.DATE }}</td>
+					<td>
+						{{ card?.AUTHOR_AUTHOR }}
+						<template v-if="model?.metadata?.annotations?.authors">
+							, {{ model.metadata.annotations.authors.join(', ') }}
+						</template>
+					</td>
+					<td>{{ card?.AUTHOR_EMAIL }}</td>
 					<td>{{ model?.metadata?.processed_by }}</td>
+					<td>{{ card?.AUTHOR_INST }}</td>
+					<td>{{ card?.LICENSE }}</td>
+					<td>{{ card?.COMPLEXITY }}</td>
```

Comment:
Don't use the `card` anymore as you are refreshing the model and reading those values from `model.metadata.card` now.

Reply:
`card` is just a computed version of `model.metadata.card` without the `UNKNOWN` strings

Reply:
The naming convention are not great here.